spec/requests/v1/publish_collection_spec.rb

```diff
@@ -23,6 +23,10 @@
     end
 
     context 'when using the legacy endpoint' do
+      after do
+        FileUtils.rm_rf(Settings.filesystems.stacks_root)
+      end
```

Comment:
This is unrelated to my change, but I got annoyed by this test failing when the suite was run in a certain order due to this test not cleaning up after itself.